sdk/openai/openai/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.0.0-beta.13 (2024-09-09)
+
+### Other Changes
+
+- Deprecation notice: The Azure OpenAI Assistants client library for JavaScript beta will be retired on January 14th, 2025. Please migrate to the stable OpenAI SDK for JavaScript using the [migration guide](https://aka.ms/oai/js/migrate) before that date.
```

Comment:
This one says assistants, but points to the generic migration guide link is that correct?

Reply:
Ah good eyes, fixed in https://github.com/Azure/azure-sdk-for-js/pull/31046/commits/3f2d18b40d857bd243cdc07879597a728efbadee